Lib/importlib/_bootstrap_external.py

```diff
@@ -173,8 +213,8 @@ def _path_isabs(path):
         """Replacement for os.path.isabs."""
         if not path:
             return False
-        root = _os._path_splitroot(path)[0].replace('/', '\\')
-        return len(root) > 1 and (root.startswith('\\\\') or root.endswith('\\'))
+        root = path[:3].replace('/', '\\')
+        return root.startswith('\\\\') or root.startswith(':\\', 1)
```

Comment:
This needs to handle the wider range of paths that `splitroot` can handle, SMB shares at least.

Reply:
Yes `PathCchSkipRoot` is not available on the xbox. Oh interesting why does `os.path.isabs` have no requirement to handle those?

Reply:
This code is almost correct. A UNC path is always absolute. But when checking for an absolute drive path, the path cannot start with a backslash. 

The existing code in `ntpath.isabs()` doesn't address this because it has a design flaw that's retained for compatibility. It returns true for rooted paths such as "\\spam". Such paths are relative to the drive or UNC share of the current working directory, which means they're definitely not absolute.

Reply:
I think I'd rather have a hacked up `PathCchSkipRoot` implementation than massively replicated Python code. Presumably we can seriously narrow its scope, and so it can be much simpler than all the cases that may be encountered on a desktop machine.

Reply:
@zooba, there is no need for `nt._path_splitroot()` in `_path_isabs()`. It's needed in `_path_join()`, however. I forgot that it was used in `_bootstrap_external`, so I mistakenly recommended just skipping it on Xbox.

Reply:
I reverted my changes to `_bootstrat_external` and instead implemented a simplified version of `_path_splitroot` for the xbox